joomla/joomla-cms
# [4.0] Add accessibility to stats table

### Summary of Changes
Add accessibility to stats table.


### Testing Instructions
In the "Joomla! would like your permission to collect some basic statistics." box, click `Select here to see the information that will be sent.`
Use browser's inspector and select Unique ID.


### Expected result
`<th scope="row">Unique ID</th>`


### Actual result
`<td><strong>Unique ID</strong></td>`
